opencascade_modules/aortic_valve_simulation/surface_shape.h

```diff
@@ -0,0 +1,67 @@
+/* -------------------------------------------------------------------------*
+ *								SPHinXsys									*
+ * -------------------------------------------------------------------------*
+ * SPHinXsys (pronunciation: s'finksis) is an acronym from Smoothed Particle*
+ * Hydrodynamics for industrial compleX systems. It provides C++ APIs for	*
+ * physical accurate simulation and aims to model coupled industrial dynamic*
+ * systems including fluid, solid, multi-body dynamics and beyond with SPH	*
+ * (smoothed particle hydrodynamics), a meshless computational method using	*
+ * particle discretization.													*                                                                        *
+ * ------------------------------------------------------------------------*/
+
+
+#ifndef SURFACE_SHAPE_H
+#define SURFACE_SHAPE_H
+
+#include "base_geometry.h"
+#include "simbody_middle.h"
+#include "opencascade/STEPCAFControl_Reader.hxx"
+#include "opencascade/XSControl_WorkSession.hxx"
+#include"opencascade/Transfer_TransientProcess.hxx"
+#include"opencascade/TopExp_Explorer.hxx"
+#include"opencascade/TopoDS.hxx"
+#include"opencascade/BRep_Builder.hxx"
+#include"opencascade/GeomAPI_ProjectPointOnSurf.hxx"
+#include"opencascade/ProjLib_ProjectOnSurface.hxx"
```

Comment:
You don't need all those headers here. Most should be in the `.cpp` You only need the header for `Standard_Real` and `Standard_CString`

Reply:
> You don't need all those headers here. Most should be in the `.cpp` You only need the header for `Standard_Real` and `Standard_CString`

Yes perhaps I need to summarize them

Reply:
> You don't need all those headers here. Most should be in the `.cpp` You only need the header for `Standard_Real` and `Standard_CString`

I have tried use them, But these two headers can't include the directory, and there is no generalised header that can include all the headers.

Reply:
The header file `surface_shape.h` only use the two typedef. You just need the OpenCascade header than define those. which seems to be `Standard_TypeDef.hxx`. The rest only needs to be in the `.cpp`

Reply:
> The header file `surface_shape.h` only use the two typedef. You just need the OpenCascade header than define those. which seems to be `Standard_TypeDef.hxx`. The rest only needs to be in the `.cpp`

ok i will change this

Reply:
I just noticed below there is also `Handle_Geom_Surface` in the class, so you need the relevant header, likely something like `Handle_Geom_Surface.hxx`

Reply:
> I just noticed below there is also `Handle_Geom_Surface` in the class, so you need the relevant header, likely something like `Handle_Geom_Surface.hxx`

ok i will change it